core/templates/dev/head/components/private_info_tooltip/private_info_tooltip_directive.html

```diff
@@ -0,0 +1,37 @@
+<div class="conversation-skin-tutor-card-tooltip">
+  <button class="conversation-skin-tutor-card-tooltip-button">?
+    <span class="conversation-skin-tutor-card-tooltip-text" translate="I18N_TUTOR_CARD_TOOLTIP"></span>
+  </button>
+</div>
+
+<style>
```

Comment:
We generally place `<style>` tag before any other tags. Ditto everywhere else.

Reply:
Ok, done!

Reply:
You forgot to do it in this file.

Reply:
Done!